spec/system/admin/user_spec.rb

```diff
@@ -8,6 +8,34 @@
   context 'when a super admin is logged in' do
     include_context 'log in super admin'
 
+    describe 'active_scaffold list', :js do
```

Comment:
Minor nit - as this is a system spec, let's focus on the behavior of this page rather than the technical underpinnings, i.e. `describe 'Users list'`...`